server/tools/cli/databases/mariadb/change-database.cli

```diff
@@ -6,4 +6,6 @@
 /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=background-validation, value=true)
 /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=background-validation-millis, value=60000)
 /subsystem=datasources/data-source=KeycloakDS: write-attribute(name=flush-strategy, value=IdleConnections)
-/subsystem=datasources/jdbc-driver=mariadb:add(driver-name=mariadb, driver-module-name=org.mariadb.jdbc, driver-xa-datasource-class-name=org.mariadb.jdbc.MySQLDataSource)
+
+/subsystem=datasources/jdbc-driver=mariadb: remove()
```

Comment:
jboss-cli supports conditions, so instead of remove/add you can just check if it's already there.

Reply:
Yeah, I read about this feature after I commited this. Wanted to use "if" here, but it doesn't work. Maybe you can help me?
When I write
```
if (outcome == success) of /subsystem=datasources/jdbc-driver=postgresql:read-resource()
  /subsystem=datasources/jdbc-driver=postgresql: remove()
end-if
```
I see this in keycloak log
```
The batch failed with the following error: : Failed to create batch from /opt/jboss/tools/cli/databases/postgres/change-database.cli: java.util.concurrent.ExecutionException: org.jboss.as.cli.CommandFormatException: The command is not available in the current context (e.g. required subsystems or connection to the controller might be unavailable).: org.jboss.as.cli.CommandFormatException: The command is not available in the current context (e.g. required subsystems or connection to the controller might be unavailable).: The command is not available in the current context (e.g. required subsystems or connection to the controller might be unavailable).
```

Reply:
I totally agree with @stianst here. I think the whole script should be conditional (not just adding database driver).

Reply:
I agree about running scripts, that rely only on configuration from environment variables, which cannot be changed for existing container.
But I don't think that doing this for scripts based on mutable configuration from volumes (like x509-related script or custom cli-scripts) is a valid approach.